ci/do_ci.sh

```diff
@@ -8,22 +8,21 @@ set -e
 echo "building using ${NUM_CPUS} CPUs"
 
 function bazel_release_binary_build() {
-  cd "${ENVOY_CONSUMER_SRCDIR}"
   echo "Building..."
-  bazel --batch build ${BAZEL_BUILD_OPTIONS} -c opt @envoy//source/exe:envoy-static.stripped.stamped
+  cd "${ENVOY_BUILD_DIR}"
```

Comment:
OK, I don't get it.

It seems that removing `cd "${ENVOY_BUILD_DIR}"` should fix the issue, and indeed, calling `./bazel/get_workspace_status` from here yields `Clean` status, but the `@envoy//source/exe:envoy-static.stripped.stamped` binary still ends up having `Modified` build-id.

Running the same command from outside Docker results in `Clean` build-id.

Any ideas?